.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest, windows-latest, macos-latest]
-        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1']
+        php-versions: ['8.1', '8.2', '8.3', '8.4']
```

Comment:
Would we want to bump the version the lint workflow is using?

Reply:
Sure, I'll do that. Thanks!